projects/Mallard/package.json

```diff
@@ -70,6 +70,7 @@
         "react-native-gesture-handler": "^1.5.3",
         "react-native-htmlview": "^0.15.0",
         "react-native-iap": "^3.3.9",
+        "react-native-image-zoom-viewer": "^3.0.1",
```

Comment:
It looks like we are only using ImageZoom, maybe we can directly import that library instead?
https://www.npmjs.com/package/react-native-image-pan-zoom

Reply:
Good point, I'll update that